06-security.Rmd

```diff
@@ -96,7 +96,27 @@ In summary, be sure to separate "trusted" from "untrusted" objects in your API i
 
 ## Cross-Origin Resource Sharing (CORS)
 
-// TODO CORS
+Cross-origin resource sharing (CORS) is a mechanism that allows restricted resources (e.g. fonts) on a web page to be requested from another domain outside the domain from which the first resource was served. A web page may freely embed cross-origin images, stylesheets, scripts, iframes, and videos. Certain "cross-domain" requests, notably Ajax requests, are forbidden by default by the same-origin security policy.
```

Comment:
I don't think we'll be able to crib this directly from Wikipedia. We'd probably need to write our own introduction.

We can just leave this as a `// TODO` if you'd like.